.bazelci/tests.yml

```diff
@@ -55,6 +55,7 @@ ubuntu1804: &ubuntu
   build_targets:
     - "//distro:distro"
     - "//distro:relnotes"
+    - "//docs:*"
```

Comment:
Needs to be `//doc_build:*` or wherever it ends up.  Seems to be breaking CI.

Reply:
Thanks for catching that.